src/components/Content/RightPanel/TaskList/TaskList.js

```diff
@@ -253,6 +260,12 @@ class TaskList extends Component {
               autoFocus
               onChange={this.onChangeFilter}
             />
+            <Button 
+              className={taskListFilter.length > 0 ? 'clear-input' : 'clear-input clear-input--hide'}
+              onClick={ this.clearFilter }
+            >
+              <FaTimes size={28} />
+            </Button>
```

Comment:
@4xdk Put this button just to the right of the text input (rather than floating over it), so that when the filter text is long, we don't have the issue of the text behind behind the button, know what I mean?